### DISCUSSION TITLE
Docs search

### DISCUSSION THREAD

**Discussion by lishaduck** [AndreaPontrandolfo/sheriff#303] [Ideas]
The blog posts mention a future "ask ai" button the website. May I suggest switching the docs search to [Orama](https://docs.orama.com/open-source/plugins/plugin-docusaurus#docusaurus-plugin)? Orama has an AI bot optionally included in the search bar. It's what the Deno team uses, and I've found it to be rather effective.

**Comment by AndreaPontrandolfo** [comment]
Oh, i'm familiar with Orama, the founder is a fellow countryman of mine and i followed him on Linkedin for some time!

Buuut... in order to get Algolia Docsearch i had to apply to their program and stuff, i don't want to risk of losing the free OSS Algolia support for Orama.
Orama is very new, it could fail tomorrow and shut down all their servers and then we would be without search lol.

Algolia is market leader, i trust their service and it just works, i don't want to risk this.

But if you have other ideas to get the "Ask AI" feature, let me know.
